autoload/leetcode.py

```diff
@@ -132,9 +133,12 @@ def is_login():
 def signin(username, password):
     global session
     session = requests.Session()
-    res = session.get(LC_LOGIN)
+    if LC_BASE.find("cn"):
```

Comment:
Single quotes could be better here.

And to check if a string contains a substring, the Pythonic way is `'cn' in LC_BASE`. And it's also worth noting that the return value is `-1` if the substring is not found.

Reply:
Thank you very much for your patience.